docs/tutorial.md

```diff
@@ -737,3 +731,90 @@ Note that even though adding type annotations to .lua files makes it invalid
 Lua, you can still do so and load them from the Lua VM once the Teal package
 loader is installed by calling tl.loader().
 
+
+### Type definitions for third party libraries
+
+You can also create declaration files to annotate the types of third-party Lua
+modules, including C Lua modules. To do so, create a file with the .d.tl
+extension and require it as normal, i.e. `require("minetest")`.
```

Comment:
It would probably be better to pick some non-global module as a first example, such as `local lfs = require("lfs")`.

Reply:
Good call, I just grabbed the one out the issue that discussed it!